project/Build.scala

```diff
@@ -57,13 +57,29 @@ object Scrooge extends Build {
       "sonatype-public" at "https://oss.sonatype.org/content/groups/public"
     ),
 
-    ScoverageSbtPlugin.ScoverageKeys.coverageHighlighting := true,
+    ScoverageKeys.coverageHighlighting := true,
 
     libraryDependencies ++= Seq(
-      "org.scalatest" %% "scalatest" % "2.2.4" % "test",
-      "org.scalacheck" %% "scalacheck" % "1.12.2" % "test",
+      scalatestLib(scalaVersion.value) % "test",
+      scalacheckLib(scalaVersion.value) % "test",
       "junit" % "junit" % "4.12" % "test"
     ),
+
+    libraryDependencies := {
```

Comment:
nice find! going to try this out. if it works we can apply the same strategy to util and the rest that need the upgrade.